source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py

```diff
@@ -0,0 +1,229 @@
+
+class JourneyDebugSerializer(PbNestedSerializer):
+    streetnetwork_duration = jsonschema.Field(schema_type=int, display_none=True, attr='sn_dur',
+                                              description='Total duration of streetnetwork use (seconds)')
+    transfer_duration = jsonschema.Field(schema_type=int, display_none=True, attr='transfer_dur',
+                                         description='Total duration of transfers (seconds)')
+    min_waiting_duration = jsonschema.Field(schema_type=int, display_none=True, attr='min_waiting_dur',
+                                            description='Minimum on all waiting durations (seconds)')
+    nb_vj_extentions = jsonschema.Field(schema_type=int, display_none=True,
+                                        description='Number of stay-in')
+    nb_sections = jsonschema.Field(schema_type=int, display_none=True,
+                                   description='Number of sections')
+    internal_id = jsonschema.Field(schema_type=str, display_none=False)
+
+
+class NoteSerializer(PbNestedSerializer):
+    #TODO
+    pass
+
+
+class ExceptionSerializer(PbNestedSerializer):
+    #TODO
+    pass
+
+
+class PathSerializer(PbNestedSerializer):
+    length = RoundedField(display_none=True)
+    name = jsonschema.Field(schema_type=str, display_none=True)
+    duration = RoundedField(display_none=True)
+    direction = jsonschema.Field(schema_type=int, display_none=True)
+
+
+class SectionSerializer(PbNestedSerializer):
+    id = jsonschema.Field(schema_type=str)
+    duration = jsonschema.Field(schema_type=int, display_none=True,
+                                description='Duration of the section (seconds)')
+    co2_emission = AmountSerializer()
+    transfer_type = EnumField()
+    departure_date_time = DateTimeField(attr='begin_date_time',
+                                        description='Departure date and time of the section')
+    arrival_date_time = DateTimeField(attr='end_date_time',
+                                      description='Arrival date and time of the section')
+    base_departure_date_time = DateTimeField(attr='base_begin_date_time',
+                                             description='Base-schedule departure date and time of the section')
+    base_arrival_date_time = DateTimeField(attr='base_end_date_time',
+                                           description='Base-schedule arrival date and time of the section')
+    to = jsonschema.MethodField(schema_type=lambda: PlaceSerializer(), deprecated=True, attr='destination')
+    def get_to(self, obj):
+        if obj.HasField(str('type')):
+            enum = obj.DESCRIPTOR.fields_by_name['type'].enum_type.values_by_number
+            ret_value = enum[getattr(obj, 'type')].name
+            if ret_value == 'WAITING':
+                return None
+        return PlaceSerializer(obj.destination).data
+
+    _from = jsonschema.MethodField(schema_type=lambda: PlaceSerializer(), deprecated=True,
```

Comment:
I guess this is a bad copy-paste from journey where they are deprecated (never filled in pb)